eZ/Publish/Core/REST/Common/Tests/FieldTypeProcessor/RichTextProcessorTest.php

```diff
@@ -61,11 +62,51 @@ public function testPostProcessFieldSettingsHash( $outputSettings, $inputSetting
         );
     }
 
+    public function testPostProcessValueHash()
+    {
+        $processor = $this->getProcessor();
+
+        $outputValue = array(
+            "xml" => <<<EOT
+<?xml version="1.0" encoding="UTF-8"?>
+<section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:ezxhtml="http://ez.no/xmlns/ezpublish/docbook/xhtml" xmlns:ezcustom="http://ez.no/xmlns/ezpublish/docbook/custom" version="5.0-variant ezpublish-1.0">
+    <title>Some text</title>
+    <para>Foobar</para>
+</section>
+EOT
+        );
+        $processedOutputValue = $outputValue;
+        $processedOutputValue["xhtml5edit"] = <<<EOT
+<?xml version="1.0" encoding="UTF-8"?>
+<section xmlns="http://ez.no/namespaces/ezpublish5/xhtml5/edit">
+    <h1>Some text</h1>
+    <p>Foobar</p>
+</section>
+
+EOT;
+
+        $this->assertEquals(
+            $processedOutputValue,
+            $processor->postProcessValueHash( $outputValue )
+        );
+    }
+
     /**
      * @return \eZ\Publish\Core\REST\Common\FieldTypeProcessor\RichTextProcessor
      */
     protected function getProcessor()
     {
-        return new RichTextProcessor;
+        $stylesheetsDir = __DIR__ . "/../../../../FieldType/RichText/Resources/stylesheets";
```

Comment:
I can't help be feel concerned when I see so many hardcoded dots... we should think of a clean(er) way to handle this. But not now :-)